script.js

```diff
@@ -0,0 +1,95 @@
+const container = document.querySelector(".container");
+const resetBtn = document.querySelector(".reset");
+const gridSize = document.querySelector(".grid-size");
+const rainbowBtn = document.querySelector(".rainbow");
+const blackBtn = document.querySelector(".black");
+const eraseBtn = document.querySelector(".erase");
+let isBlack = true;
+
+rainbowBtn.addEventListener("click", () => {
+  let color = "rainbow";
+  setColor(color);
+});
+
+blackBtn.addEventListener("click", () => {
+  let color = "black";
+  setColor(color);
+});
+
+eraseBtn.addEventListener("click", () => {
+  let color = "erase";
+  setColor(color);
+});
+
+sizingGrid(16);
+creatingDivs(16);
+
+function sizingGrid(grid) {
+  container.style.setProperty("--grid-rows", grid);
+  container.style.setProperty("--grid-cols", grid);
+}
+
+gridSize.addEventListener("click", function () {
+  let size = prompt("Grid size? 0-64");
```

Comment:
What if the user enters a random string that's not a number? Like `abracadabra`. Maybe you would want to convert the input to a number and confirm it's actually a number.